setup.cfg

```diff
@@ -1,2 +1,47 @@
 [metadata]
 license_file = LICENSE.txt
+
+[flake8]
+ignore =
```

Comment:
you can use full names instead of shortcuts

Reply:
??? How?

Reply:
my bad, it seems that it works only for pylint